src/lib/shmem/src/lib.rs

```diff
@@ -18,3 +18,5 @@ mod c_bindings {
     #![allow(improper_ctypes)]
     include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 }
+
+pub mod scmutex;
```

Comment:
nit: put this above next to `pub mod allocator`?

Reply:
done